codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -332,15 +373,23 @@ protected void doWrite(Object msg) {
                 ReferenceCountUtil.release(msg);
                 throw new IllegalArgumentException("Message must be an Http2StreamFrame: " + msg);
             }
-
             Http2StreamFrame frame = (Http2StreamFrame) msg;
-            if (frame.streamId() != -1) {
+            ChannelPromise promise = ctx.newPromise();
+            if (Http2CodecUtil.isStreamIdValid(frame.streamId())) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unnecessary use of fully qualified name 'Http2CodecUtil.isStreamIdValid' due to existing static import 'io.netty.handler.codec.http2.Http2CodecUtil.isStreamIdValid' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)